components/Header.js

```diff
@@ -26,7 +26,7 @@ export default function Header() {
               News &amp; Events
             </a>
           </Link>
-          <Link href="/">
+          <Link href="/contactUs">
```

Comment:
@himanshuvarandani I forgot to mention this.
This should be `<Link href="/contact">`

Reply:
ok I will update this in another pr

Reply:
I think the email validation commit made was supposed to make this change.

Reply:
I am making a direct commit.